homeassistant/components/lutron/config_flow.py

```diff
@@ -68,3 +74,34 @@ async def async_step_user(
             ),
             errors=errors,
         )
+
+    @staticmethod
+    @callback
+    def async_get_options_flow(
+        config_entry: ConfigEntry,
+    ) -> OptionsFlowHandler:
+        """Get the options flow for this handler."""
+        return OptionsFlowHandler()
+
+
+class OptionsFlowHandler(OptionsFlow):
+    """Handle option flow for lutron."""
+
+    async def async_step_init(
+        self, user_input: dict[str, Any] | None = None
+    ) -> ConfigFlowResult:
+        """Handle options flow."""
+        if user_input is not None:
+            return self.async_create_entry(title="", data=user_input)
+
+        data_schema = vol.Schema(
+            {
+                vol.Required(
+                    CONF_DEFAULT_DIMMER_LEVEL,
+                    default=self.config_entry.options.get(
+                        CONF_DEFAULT_DIMMER_LEVEL, DEFAULT_DIMMER_LEVEL
+                    ),
+                ): vol.All(vol.Coerce(int), vol.Range(min=1, max=255)),
```

Comment:
Also, I think you can use a NumberSelector here (not sure how this currently shows in the flow)

Reply:
Now looks like this:

![Screenshot 2025-02-23 at 10 35 38](https://github.com/user-attachments/assets/a3a9a922-94ff-47a9-81f7-8ba164bad4c5)